docs/content/preview/migrate/known-issues/postgresql.md

```diff
@@ -59,3 +61,79 @@ CREATE TABLE public.sales_region (
 )
 PARTITION BY LIST (region);
 ```
+
+---
+
+### Index creation on partitions fail for some YugabyteDB builds
+
+**GitHub**: [Issue 14529](https://github.com/yugabyte/yugabyte-db/issues/14529)
+
+**Description**: If you have a partitioned table with indexes on it, the migration will fail with an error for YugabyteDB `2.15` or `2.16` due to a regression.
+
+Note that it is fixed in the release [12.17.1.0](../../../releases/release-notes/v2.17/#v2.17.1.0).
+
+**Workaround**: N/A
+
+**Example**
+
+An example schema on the source database is as follows:
+
+```sql
+DROP TABLE IF EXISTS list_part;
+
+CREATE TABLE list_part (id INTEGER, status TEXT, arr NUMERIC) PARTITION BY LIST(status);
+
+CREATE TABLE list_active PARTITION OF list_part FOR VALUES IN ('ACTIVE');
+
+CREATE TABLE list_archived PARTITION OF list_part FOR VALUES IN ('EXPIRED');
+
+CREATE TABLE list_others PARTITION OF list_part DEFAULT;
+
+INSERT INTO list_part VALUES (1,'ACTIVE',100), (2,'RECURRING',20), (3,'EXPIRED',38), (4,'REACTIVATED',144), (5,'ACTIVE',50);
+
+CREATE INDEX list_ind ON list_part(status);
+```
+
+---
+
+### Creation of certain views in the rule.sql file
+
+**GitHub**: [Issue 770](https://github.com/yugabyte/yb-voyager/issues/770)
+
+**Description**: There may be few cases where certain exported views come under the `rule.sql` file and the `view.sql` file might contain a dummy view definition. This `pg_dump` behaviour may be due to how PostgreSQL handles views internally (via rules).
+
+{{< note title ="Note" >}}
+This does not affect the migration as YugabyteDB Voyager takes care of the DDL creation sequence internally.
+{{< /note >}}
+
+**Workaround**: Not required
+
+**Example**
+
+An example schema on the source database is as follows:
+
+```sql
+CREATE TABLE foo(n1 int PRIMARY KEY, n2 int);
+CREATE VIEW v1 AS
+  SELECT n1,n2
+  FROM foo
+  GROUP BY n1;
+```
+
+The exported schema for `view.sql` is as follows:
```

Comment:
Should this be rule.sql? they are both view.sql currently

Reply:
yes , thanks